project/src/components/sort/sort.tsx

```diff
@@ -1,30 +1,49 @@
+import clsx from 'clsx';
+import { useState } from 'react';
+import { useAppDispatch, useAppSelector } from '../../hooks';
+import { SortType } from '../../const';
+import { changeSort, updateOffers } from '../../store/actions';
 
 export default function Sort(): JSX.Element {
+
+  const [isShowSort, setIsShowSort] = useState<boolean>(false);
```

Comment:
У примитивов ts отлично выводит тип сам можно опустить
```suggestion
  const [isShowSort, setIsShowSort] = useState(false);
```